opentracing/src/main/java/opentracing/Span.java

```diff
@@ -57,24 +57,28 @@
   /** Get a Baggage item by key. */
   String getBaggageItem(String key);
 
-  /**   *
-   * Add a new log event to the Span, accepting an event name string and an optional structured payload argument.
-   * If specified, the payload argument may be of any type and arbitrary size,
-   * though implementations are not required to retain all payload arguments
-   * (or even all parts of all payload arguments).
-   *
-   * The timestamp of this log event is the current time.
-   **/
-  Span log(String eventName, /* @Nullable */ Object payload);
-
   /**
-   * Add a new log event to the Span, accepting an event name string and an optional structured payload argument.
-   * If specified, the payload argument may be of any type and arbitrary size,
-   * though implementations are not required to retain all payload arguments
-   * (or even all parts of all payload arguments).
+   * Create an event builder, specifying the mandatory event name string.
    *
-   * The timestamp is specified manually here to represent a past log event.
-   * The timestamp in microseconds in UTC time.
+   * The timestamp of this log event is by default the time the EventBuilder is constructed.
    **/
-  Span log(long instantMicroseconds, String eventName, /* @Nullable */ Object payload);
+  EventBuilder buildEvent(String eventName);
```

Comment:
requiring an `eventName` makes this a bad fit for info/error debug logging... we could still have `buildEvent()` as a sibling to info/error, though I imagine you're not a fan of that...